### DISCUSSION TITLE
Is it possible to force glyphs to overwrite existing code points in a font?

### DISCUSSION THREAD

**Discussion by joshfrench** [ryanoasis/nerd-fonts#1132] [General]
Source Code includes a few janky glyphs of its own: 

<img width="138" alt="Screenshot 2023-02-22 at 18 16 00" src="https://user-images.githubusercontent.com/4573/220785152-4654f4cd-1d2d-4846-9979-cd0da339068d.png">

There are others, these are just the two that vex me the most. I'd like to use the built-in system emoji for these instead. I'm not even sure that this is something nerd-fonts can do as these are probably outside the code points that the font patcher cares about, but I thought I'd ask just in case. Thanks!

**Comment by Finii** [comment]
The `font-patcher` overwrites (almost) always existing glyphs unless you say `--careful`.

You could rip the glyphs out of the system's emoji font and use them together with the `--custom` option to patch them in.
An example script to rip glyphs out is https://github.com/ryanoasis/nerd-fonts/blob/-/bin/scripts/generate-extraglyphs.py

Or you can open the font in `fontforge` und just delete the two codepoints.

**Reply by joshfrench** [reply]
Oh neat, I’ll take a look at those approaches. Thanks so much!
